src/test/java/space/obminyashka/items_exchange/controller/UserControllerIntegrationTest.java

```diff
@@ -140,6 +140,16 @@ void updateUserPassword_WhenOldPasswordWrong_ShouldThrowInvalidDtoException() th
         assertTrue(mvcResult.getResponse().getContentAsString().contains(getMessageSource("incorrect.password")));
     }
 
+    @Test
+    @WithMockUser(username = "user")
+    void updateUserPassword_WhenUserEnteredOldPassword_ShouldThrowIllegalArgumentException() throws Exception {
```

Comment:
Small changes need to be introduced to the method's name: when and should words have to start with a low letter because of the camel case style.